makefiles/boards/sam0.inc.mk

```diff
@@ -24,8 +24,15 @@ DEBUG_ADAPTER ?= dap
 
 # EDBG can only be used with a compatible Atmel programmer
 ifeq ($(DEBUG_ADAPTER),dap)
-  # set this to either openocd or edbg
+  # set this to either openocd, jlink or edbg
   PROGRAMMER ?= edbg
+else ifeq ($(DEBUG_ADAPTER),jlink)
+  # only use JLinkExe if it's installed
+  ifneq (,$(shell which JLinkExe)))
```

Comment:
Please use `command -v` instead of `which` for portability.

Reply:
Ok.